Thinktecture.Relay.OnPremiseConnector/SignalR/HeartbeatChecker.cs

```diff
@@ -17,20 +17,7 @@ public void Check(IRelayServerConnection connection)
 			if (connection == null)
 				throw new ArgumentNullException(nameof(connection));
 
-			var logger = _logger?
-				.ForContext("RelayServerUri", connection.Uri)
-				.ForContext("RelayServerConnectionInstanceId", connection.RelayServerConnectionInstanceId);
-
-			var lastHeartbeat = connection.LastHeartbeat;
-			if (lastHeartbeat != DateTime.MinValue && lastHeartbeat != DateTime.MaxValue)
```

Comment:
In the moved method, I removed the checked `lastHeartbeat != DateTime.MaxValue` as there is no place in the code the `LastHeartbeat` will be set to `DateTime.MaxValue`